frontend/package.json

```diff
@@ -58,7 +59,7 @@
     "vue-jest": "^3.0.7",
     "vue-loading-overlay": "^3.4.2",
     "vue-moment": "^4.1.0",
-    "vue-qrcode": "^0.3.5",
+    "vue-qrcode": "0.3.5",
```

Comment:
do we need this library?

Reply:
im PR https://github.com/gradido/gradido/pull/1764 habe ich dies aufgeräumt.